--- setup.cfg.orig	2010-04-07 05:32:19.000000000 +0200
+++ setup.cfg	2010-04-07 05:32:41.000000000 +0200
@@ -1,9 +1,9 @@
 [_ldap]
 extra_objects = 
 extra_compile_args = 
-libs = ldap_r lber sasl2 ssl crypto
-library_dirs = /opt/openldap-RE24/lib
-include_dirs = /opt/openldap-RE24/include /usr/include/sasl
+libs = ldap_r lber sasl2 ssl crypto m
+library_dirs =
+include_dirs =
 
 [egg_info]
 tag_build = 
